src/System.IO.Pipelines/src/System/IO/Pipelines/Pipe.cs

```diff
@@ -75,8 +75,8 @@ public sealed partial class Pipe
 
         // The write head which is the extent of the PipeWriter's written bytes
         private BufferSegment _writingHead;
-        private Memory<byte> _writingMemory;
-        private int _buffered;
+        private Memory<byte> _writingHeadMemory;
+        private int _writingHeadBytesBuffered;
```

Comment:
I renamed these just to be consistent with the other implementation. I can revert if this causes the diff to be confusing.